dot/core/messages_test.go

```diff
@@ -137,7 +137,7 @@ func TestService_HandleTransactionMessage(t *testing.T) {
 	ks.Acco.Insert(kp)
 
 	bp := new(MockBlockProducer) // nolint
-	blockC := make(chan types.Block)
+	blockC := make(chan types.Block, 100)
```

Comment:
not sure if buffered channels are needed in tests @arijitAD  what do you think?

Reply:
I wasn't sure if I needed to add them to tests, but I thought I wouldn't hurt....

Reply:
Yes. We can keep them unbuffered in tests.